src/main/java/ro/msg/learning/shop/dto/ProximityLocationDTO.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import ro.msg.learning.shop.entities.Address;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ProximityLocationDTO {
+    private String cityState;
+
+
+    public static ProximityLocationDTO of(Address address){
+
+        return ProximityLocationDTO.builder()
+                .cityState(address.getCity()+","+" "+address.getState())
```

Comment:
you can also use a StringBuilder instead of appending strings
https://dev.to/kylec32/effective-java-beware-the-performance-of-string-concatenation-5e7l

Reply:
Using "**StringBuilder**" instead of appending strings